pkg/controller/controller_test.go

```diff
@@ -1471,6 +1431,7 @@ func runProvisionTest(t *testing.T, k string, tc provisioningTestcase, requested
 	if err != nil {
 		t.Fatal(err)
 	}
+	mockTranslator := NewMockProvisionerCSITranslator(mockController)
```

Comment:
Does this test do anything with `mockTranslator`? If not, can we pass in a real translator? (assuming the test passes fine with it.)

Reply:
nope, changing all other tests to use real translator